Source/ViewerController.swift

```diff
@@ -2,6 +2,18 @@ import UIKit
 import CoreData
 
 /**
+ Describe the index path as a simple string, so we can use it as a key for our queueing dictionary
+ */
+extension IndexPath {
+    var indexPathString: String {
+        get {
+            return String((self as NSIndexPath).section) + "-" + String((self as NSIndexPath).row)
```

Comment:
For this particular case, yes. But I just imported some code I had from `<redacted>`. 🙈